magento/magento2
# Selecting Existing Address in Admin Order

### Description
Selecting an existing address while creating an order does not have the customer_address_id field set correctly so this data is not passed to backend and saved. On page refresh 'Add New' is still selected. 

The existing code tries to set this value but does so incorrectly so it is not saved.

### Manual testing scenarios
1. Create a new order for an existing customer that has at least one billing address saved
2. Select 'Add New' from the billing address dropdown and refresh the page (Add New should be selected)
3. Select an existing address from the billing address dropdown and refresh the page. Add New is still selected with corret billing address in indivual inputs.

### Questions or comments
There may be a better way to resolve this issue, but this is what I did in my project (although I extended the method and added the change there).


### Resolved issues:
1. [x] resolves magento/magento2#32906: Selecting Existing Address in Admin Order
